packages/measure/src/measure-renders.tsx

```diff
@@ -35,17 +35,28 @@ export async function measureRender(ui: React.ReactElement, options?: MeasureOpt
   const warmupRuns = options?.warmupRuns ?? config.warmupRuns;
 
   const { render, cleanup } = resolveTestingLibrary();
+  const testingLibrary = getTestingLibrary();
 
   showFlagsOutputIfNeeded();
 
   const runResults: RunResult[] = [];
   let hasTooLateRender = false;
+  let hasUnnecessaryRender = false;
```

Comment:
There seems to be no code chaning this value. 

I would suggest renaming this to (+ change type to number)
```
let redundantIntialRendersCount = 0'
```